src/services/earn/types.ts

```diff
@@ -198,6 +199,12 @@ export enum StrategyRiskLevel {
   HIGH = 'HIGH',
 }
 
+export enum StrategyStatus {
+  OK = 'OK',
+  RESCUE_NEEDS_CONFIRMATION = 'RESCUE_NEEDS_CONFIRMATION',
+  RESCUED = 'RESCUED',
```

Comment:
Let's update the values as in https://github.com/Balmy-protocol/api/pull/679#pullrequestreview-2576025369